trivia-game/client/src/components/header.js

```diff
@@ -0,0 +1,20 @@
+const Header = (props) => {
+
+  let message1;
+  let message2;
+  if (props.user) {
+    message1 = `WELCOME TO TANYA'S GAMEBOI`;
+    message2=  `Game on, ${props.user}!`;
+  } else {
+    message1 = `WELCOME TO TANYA'S GAMEBOI`;
```

Comment:
Because `message1` does not change, it could be declared outside of the conditional. (i.e. replace line three with `const message1 = "WELCOME TO TANYA'S GAMEBOI";